test/bdd/steps/network.js

```diff
@@ -356,6 +357,29 @@ Then(/^the last exported dataset signature should belong to ([DC|DV]+)$/, async
     expect(utilities.verifySignature(response.data.formatted_dataset, myNode.options.nodeConfiguration.node_wallet), 'Signature not valid!').to.be.true;
 });
 
+Then(/^the last exported dataset contains imported data$/, async function () {
+    expect(this.state.nodes.length, 'No started nodes').to.be.greaterThan(0);
+    expect(!!this.state.lastExportHandler, 'Last export didn\'t happen. Use other step to do it.').to.be.equal(true);
+
+    const ot_logo = utilities.base64_encode(path.resolve(__dirname, '../../../importers/xml_examples/Retail/ot_logo.svg'));
```

Comment:
You assume this is the imported data in the step, but the step only states "imported data". 
Looking at the scenario you should either retrieve the data from the `lastImport`, or you can specify which data should be contained in the step text as a variable, similar to path that gets specified in the import step.